macOS/DuckDuckGo/TabBar/View/SpinnerView.swift

```diff
@@ -96,14 +100,20 @@ extension SpinnerView {
     }
 
     func stopAnimating(animated: Bool = true) {
```

Comment:
This API will now be invoked more than once: when the `Page DOM Loaded` event fires, and when `isLoading` signals `false`.

These changes are meant to prevent UI glitches derived from multiple subsequent `stopAnimating` invocations